### ISSUE TITLE
v3.4.2 throws a TypeError when parsing "@import url(...)"

### ISSUE THREAD

**Issue by shinnn** [jakubpawlowicz/clean-css#668]
### Versions

```
$ node -v
v4.0.0
$ npm ls clean-css
/Users/Shinnosuke/test
└── clean-css@3.4.2 
```

### Script

./example.js

```js
'use strict';

var CleanCSS = require('clean-css');

var css = '@import url(https://fonts.googleapis.com/css?family=Open+Sans);';
new CleanCSS().minify(css, function() {});
```

### Result

```
$ node example.js
path.js:8
    throw new TypeError('Path must be a string. Received ' +
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.join (path.js:477:5)
    at relative (/Users/Shinnosuke/test/node_modules/clean-css/lib/urls/rewrite.js:43:45)
    at rebase (/Users/Shinnosuke/test/node_modules/clean-css/lib/urls/rewrite.js:71:15)
    at /Users/Shinnosuke/test/node_modules/clean-css/lib/urls/rewrite.js:88:44
    at byUrl (/Users/Shinnosuke/test/node_modules/clean-css/lib/urls/reduce.js:72:5)
    at reduceAll (/Users/Shinnosuke/test/node_modules/clean-css/lib/urls/reduce.js:145:10)
    at rewriteUrls (/Users/Shinnosuke/test/node_modules/clean-css/lib/urls/rewrite.js:86:10)
    at IncomingMessage.<anonymous> (/Users/Shinnosuke/test/node_modules/clean-css/lib/imports/inliner.js:311:24)
    at emitNone (events.js:72:20)
```

**Comment by chrisjstephens** [comment]
I got the same error.

**Comment by rubikzube** [comment]
Moi aussi

**Comment by jeffwhelpley** [comment]
I get this error as well

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.3.
